olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/Types.h

```diff
@@ -139,6 +140,12 @@ using VersionsResponseCallback = Callback<VersionsResult>;
 
 /// The list of tile keys.
 using TileKeys = std::vector<geo::TileKey>;
+
+/// An alias for the compatible versions response.
+using CompatibleVersionsResponse = Response<CompatibleVersionsResult>;
+/// The compatible versions request callback.
+using CompatibleVersionsCallback = Callback<CompatibleVersionsResult>;
```

Comment:
Please group relevant aliases together. Means this should not be after TileKeys but after `VersionsResponseCallback`.